broker/fsm.go

```diff
@@ -1,9 +1,12 @@
 package broker
 
 import (
+	"encoding/json"
 	"io"
 
 	"github.com/hashicorp/raft"
```

Comment:
I had thought of doing that but this change set was already big for a PR, so I was planning to come back later for it. I'll take your suggestion and make the change now. I'll put a wrapper around serf.MemberStatus and raft.fsm. I'll update here when done.